config/src/main/kotlin/maru.config/Config.kt

```diff
@@ -73,7 +96,7 @@ data class MaruConfig(
   val persistence: Persistence,
   val sotNode: ApiEndpointConfig,
   val qbftOptions: QbftOptions,
-  val p2pConfig: P2P?,
+  val p2pConfig: P2P,
```

Comment:
I believe we agreed that it should be nullable after all. So we don't have to bother with P2P when we don't need it in some tests